src/vivarium_gates_nutrition_optimization/components/hemoglobin.py

```diff
@@ -189,21 +206,53 @@ def sample_from_hemoglobin_distribution(
         ret_val.loc[gumbel] = self._mirrored_gumbel_ppf_2017(propensity, mean, sd)[gumbel]
         return ret_val
 
-    # def adjust_maternal_hemorrhage_probability(self, index, probability):
-    #     paf = self.hemorrhage_paf(index)["value"]
-    #     rr = self.hemorrhage_rr(index)["value"]
-    #     p_maternal_hemorrhage = probability["moderate_maternal_hemorrhage"] + probability["severe_maternal_hemorrhage"]
-    #     severe_ratio = probability["severe_maternal_hemorrhage"] / p_maternal_hemorrhage
-    #     p_maternal_hemorrhage_nonanemic = p_maternal_hemorrhage * (1 - paf)
-    #     p_maternal_hemorrhage_anemic = p_maternal_hemorrhage_nonanemic * rr
-    #     hemoglobin = self.hemoglobin(index)
-    #     anemic = hemoglobin <= SEVERE_ANEMIA_AMONG_PREGNANT_WOMEN_THRESHOLD
-    #     probability["severe_maternal_hemorrhage"] = severe_ratio * p_maternal_hemorrhage_nonanemic
-    #     probability["moderate_maternal_hemorrhage"] = (1 - severe_ratio) * p_maternal_hemorrhage_nonanemic
-    #     probability.loc[anemic, "severe_maternal_hemorrhage"] = severe_ratio * p_maternal_hemorrhage_anemic
-    #     probability.loc[anemic, "moderate_maternal_hemorrhage"] = (1 - severe_ratio) * p_maternal_hemorrhage_anemic
-    #     probability["not_maternal_hemorrhage"] = (1 - probability["moderate_maternal_hemorrhage"] - probability["severe_maternal_hemorrhage"])
-    #     return probability
+    def adjust_maternal_disorder_proportion(
```

Comment:
the map to truncate to one is new, but otherwise the same.

Reply:
Is it expected that the adjusted proportion could be greater than 1, or is this masking a bug?

Reply:
I'm not sure why IV iron didn't do this, but the [documentation](https://vivarium-research.readthedocs.io/en/latest/models/other_models/hemoglobin_anemia_and_iron_deficiency/risk_effects/index.html#maternal-hemorrhage-incidence) specifies that it is possible to be greater than one, yeah